test/runtime-integration/webpack-5/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@babel-internal/runtime-integration-webpack-5",
+  "private": true,
+  "exports": "./index.js",
+  "devDependencies": {
+    "@babel/runtime": "workspace:*",
```

Comment:
Can you duplicate cases to `@babel/runtime-corejs3`? We have extra core-js imports in that package.